mantidproject/mantid
# Change some logs from LoadWANDSCD to be TimeSeriesProperties

This will allow them to work correctly with SetGoniometer. Differences in testing output are just due to small floating point differences.

Previously these logs were stored as double vectors which don't work with SetGoniometer


**To test:**

This should run successfully

```python
ws = LoadWANDSCD('HB2C_7000.nxs.h5,HB2C_7001.nxs.h5')
SetGoniometer(ws, Axis0='s1,0,1,0,1')
```

on master it fails with

```
SetGoniometer-[Error] Error in execution of algorithm SetGoniometer:
SetGoniometer-[Error] Run::getPropertyAsSingleValue - Property "s1" is not a single numeric value or numeric time series.
ValueError: Run::getPropertyAsSingleValue - Property "s1" is not a single numeric value or numeric time series.
```

The logs should also be plottable in the sample logs viewer now.

*There is no associated issue.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
